doc/plugin_server_bundlepublisher_k8s_configmap.md

```diff
@@ -0,0 +1,57 @@
+# Server plugin: BundlePublisher "k8s_configmap"
+
+The `k8s_configmap` plugin puts the current trust bundle of the server in a designated
+Kubernetes ConfigMap, keeping it updated. The plugin supports configuring multiple clusters.
+
+The plugin accepts the following configuration options:
+
+| Configuration   | Description                                                                                                                                                                                            | Required                                    | Default |
+|-----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------|---------|
+| configmap_name  | The name of the ConfigMap.                                                                                                                                                                             | Yes.                                        |         |
+| configmap_key   | The key within the ConfigMap for the bundle.                                                                                                                                                           | Yes.                                        |         |
+| namespace       | The namespace containing the ConfigMap.                                                                                                                                                                | Yes.                                        |         |
+| kubeconfig_path | The path on disk to the kubeconfig containing configuration to enable interaction with the Kubernetes API server. If unset, in-cluster credentials will be used. Only one in-cluster can be configured.| Required when configuring a remote cluster. |         |
+| format          | Format in which the trust bundle is stored, &lt;spiffe &vert; jwks &vert; pem&gt;. See [Supported bundle formats](#supported-bundle-formats) for more details.                                         | Yes.                                        |         |
```

Comment:
These should be nested in the `clusters` structure. Maybe we should specify it similarly to [k8s_psat](https://github.com/spiffe/spire/blob/main/doc/plugin_server_nodeattestor_k8s_psat.md)?